static/src/stylesheets/layout/_ab-new-header-test-variant.scss

```diff
@@ -715,61 +715,327 @@ $navigation-horizontal-padding: $gs-gutter * 2 + $gs-gutter/2;
+
+            @include mq(mobileLandscape) {
+                height: $gutter-large + $veggie-burger-medium - $gs-gutter/2;
+                width: $gutter-large + $veggie-burger-medium - $gs-gutter/2;
+                top: -50px;
+            }
+        }
+    }
+}
+
 .tertiary-navigation {
+    margin: 0;
+    background-color: $neutral-2;
+    padding-left: $gs-gutter/2;
+    padding-right: $gs-gutter/2;
     list-style: none;
+    white-space: nowrap;
     display: flex;
-    flex-direction: row;
-    flex-flow: row wrap;
-    justify-content: space-between;
-    margin-bottom: -$gs-baseline/2;
-    margin-left: $gs-gutter / 2;
-    margin-right: $gs-gutter / 2;
 
-    @include mq(mobileLandscape) {
-        margin-left: $gs-gutter;
-        margin-right: $gs-gutter;
+    // Scrolly variant
+    .tertiary__variant-three & {
+        overflow-x: scroll;
+        padding-top: 0;
+        padding-bottom: $gs-baseline*3;
     }
-}
 
-.tertiary-navigation__item {
-    width: calc(50% - #{$gs-gutter / 2});
-    border-top: 1px solid $neutral-5;
-}
+    // Stacked variant
+    .tertiary__variant-four & {
+        padding-top: $gs-baseline/3;
+        padding-bottom: $gs-baseline/3;
+        flex-wrap: wrap;
 
-.tertiary-navigation__text {
-    width: calc(100% - #{$gs-gutter});
-    float: left;
+        @include mq($until: $mobile-medium) {
+            max-height: 40px;
+        }
+
+        @include mq($from: $mobile-medium) {
+            max-height: $gs-baseline*4.5;
+        }
+    }
+
+    @include mq($from: mobileLandscape) {
+        padding-left: $gs-gutter;
+        padding-right: $gs-gutter;
+    }
+
+    // Single line limited variant
+    .tertiary__variant-five & {
+        padding-top: 0;
+        padding-bottom: 0;
+        height: $gs-baseline * 2.5;
+        flex-wrap: wrap;
+        padding-right: $gutter-small + $veggie-burger-small;
+        overflow: hidden;
+
+        @include mq($mobile-medium) {
+            height: $gs-baseline * 3;
+            padding-right: $gutter-medium + $veggie-burger-small;
+        }
+
+        @include mq(mobileLandscape) {
+            padding-right: $gutter-large + $veggie-burger-medium;
+        }
+    }
 }
 
 .tertiary-navigation__link {
-    @include fs-textSans(5);
-    line-height: 18px;
-    font-weight: 600;
-    color: $news-main-1;
-    min-height: $gs-row-height;
-    padding-top: $gs-baseline / 3;
-    padding-bottom: $gs-baseline / 2;
+    position: relative;
+    @include fs-textSans(4);
+    color: $neutral-4;
     display: block;
+
+    @include mq($from: mobile, $until: $mobile-medium) {
+        font-size: 14px;
+    }
+
+    @include mq($mobile-medium) {
+        font-size: 18px;
+    }
+
+    &:after {
+        pointer-events: none;
+        content: '/';
+        color: #8a8a8a;
+    }
+
+    // Scrolly variant, Single line limited variant
+    .tertiary__variant-three &,
+    .tertiary__variant-five & {
+        line-height: $gs-baseline * 2.5;
+        padding: 0 .4em;
```

Comment:
In this case we want the padding to be in relation to the font-size. CC @zeftilldeath